soc/arm/nuvoton_npcx/common/soc_dt.h

```diff
@@ -18,6 +18,27 @@
 	  .bit  = DT_PHA(DT_DRV_INST(inst), clocks, bit),                      \
 	}
 
+/* Construct a npcx_clk_cfg structure from clocks property at index 'i' */
```

Comment:
Simon also reminded me in the other PR. In the beginning, I think it's unnecessary because they are only used for npcx series internally. But it seems to give reviewers a headache without detailed comment. I will add a CL to add comments to describe parameter, return type, and usage of each macro function.

Reply:
Done.